src/rules/group-exports.js

```diff
@@ -0,0 +1,133 @@
+const meta = {}
+const errors = {
+  ExportNamedDeclaration:
+    'Multiple named export declarations; consolidate all named exports into a single statement',
+  ExportDefaultDeclaration:
+    'Default export declaration should be adjacent to named export',
+  MemberExpression:
+    'Multiple CommonJS exports; consolidate all exports into a single assignment to ' +
```

Comment:
any reason this isn't on a single line? if `max-len` is complaining, let's enable the option that ignores long strings.